### PR TITLE
Add best practice tip for wallet management

### PR DESCRIPTION
**Pull request by developing-coder**
Suggest keeping investing and development wallets separate

### CODE REVIEW

File: NFT_Game/en/Section_1/Lesson_5_Deploy_To_Rinkeby.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The next step is a testnet which you can think of as like a "staging" environmen
 
 ### 🦊 Metamask
 
+**BEST PRACTICE! Create a new account in MetaMask strictly for development purposes. This way, if your development account credentials accidentally leak, your real funds will still be safe.**
```
Comment (adilanchian) [inline NFT_Game/en/Section_1/Lesson_5_Deploy_To_Rinkeby.md:9]: Can you change "BEST PRACTICE" to be - "Note:"

**Review by adilanchian — CHANGES_REQUESTED** [review]
Looks great! Just have a small ask here :)

TY!

File: NFT_Game/en/Section_1/Lesson_5_Deploy_To_Rinkeby.md (hunk 1 above)
Comment (developing-coder) [reply NFT_Game/en/Section_1/Lesson_5_Deploy_To_Rinkeby.md]: Done!

**Comment by adilanchian** [comment]
Sweet thank you!

**Pull request merged**